app/models/role.rb

```diff
@@ -37,32 +37,21 @@ def self.included(base)
       base.extend(ClassMethods)
```

Comment:
Modernised the scopes. Rails wasn't able to order the old version properly as it couldn't generate the correct SQL